serverless/containers/how-to/add-a-custom-domain-to-a-container.mdx

```diff
@@ -36,11 +36,65 @@ Each container comes with an automatically generated domain, on which it is avai
 
           - If your DNS provider does not support CNAME flattening, you need to create an ALIAS on your root domain (named `@` or with an empty name) instead of a CNAME.
     </Message>
-2. Click the **Endpoints** tab on your containers' page.
-3. Click **+ Add endpoint**.
-4. Enter your custom domain name and validate the form.
-5. The console runs checks on the CNAME and TLS configuration of the custom domain name. Connect to your container using your custom domain once all checks are complete.
+2. Before continuing, make sure the DNS record is propagated. If not, the custom domain will likely end up in `error` on step 6. You can check this by:
+  - running `dig`, `drill` or `nslookup` commands on your domain using common DNS resolvers `1.1.1.1` and `8.8.8.8`: `dig <your_custom_domain> @1.1.1.1`, `dig <your_custom_domain> @8.8.8.8`
+  - or by using the DNS check propagation tool of your choice (some are available online)
+3. Click the **Endpoints** tab on your containers' page.
+4. Click **+ Add endpoint**.
+5. Enter your custom domain name and validate the form.
+6. The console runs checks on the CNAME and TLS configuration of the custom domain name. See technical details below for more details.
+7. Once the custom domain status is `ready`, you can connect to your container through that custom domain.
 
 <Message type="important">
   **Custom Domains** do not currently support websockets. If you need to use websockets, you must configure them with the original container URL.
 </Message>
+
+## Technical details and troubleshooting
+
+Read this section if you want more details about how custom domains are handled or if you are trying to troubleshoot an issue.
+
+When adding a custom domain, the following tasks will be performed on Scaleway's side:
+
+1. Configure the custom domain on our gateways, so they know that they should handle traffic for that custom domain.
```

Comment:
quibble: simplify wording

```suggestion
1. Configure the custom domain on our gateways, so that they can handle traffic for that custom domain.
```